Library/Homebrew/test/rubocops/urls_spec.rb

```diff
@@ -183,8 +183,8 @@
     }]
   }
 
-  context "When auditing urls" do
-    it "with offenses" do
+  context "when auditing URLs" do
+    it "reports any offenses" do
```

Comment:
Minor but maybe this should be:

```suggestion
    it "reports all offenses" do
```

Reply:
I wasn't sure how to describe this as technically this doesn't report all offenses since there are the two others below this

Reply:
Yeah, I see what you mean. What you just pushed looks great to me (plus bonus clarity because `formulae` isn't that descriptive of what it is)